stock_quant_manual_assign/models/stock_picking_type.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockPickingType(models.Model):
+    _inherit = "stock.picking.type"
+
+    auto_fill_qty_done = fields.Boolean(
+        "Auto-fill Quantity Done",
+        help="Select this in case done quantity of the stock move line should "
+        "be auto-filled when quants are manually assigned.",
+    )
```

Comment:
I think this this field should be removed. This option is of no use after https://github.com/odoo/odoo/commit/7dda6bb92715ea25b2818a62fec5e646f3678b81.

Reply:
done!